hedy_content.py

```diff
@@ -348,6 +357,9 @@
     ]
 }
 
+ADVENTURE_ORDER_PER_LEVEL = {level: [a for a in adventures if a not in ['parsons', 'quiz']]
```

Comment:
This variable is used a lot across app.py and for_teachers.py, in this last file its often used as `hedy_content.ADVENTURE_ORDER_PER_LEVEL`. You also need to replace it there, becase at this moment is like the parsons and quizes were removed, for example, in the restore customizations, like Annelies was suggesting.

Reply:
Thank you very much for pointing this out. I have no clue how I managed to not replace it everywhere.